changelog/pending/20241109--auto-go-nodejs-python--fixed-a-bug-where-token-information-was-not-present-in-types.yaml

```
@@ -0,0 +1,4 @@
+changes:
+- type: fix
+  scope: auto/go,nodejs,python
+  description: Fixed a bug where token information was not present in types
```

```suggestion
  description: Expose `whoami` token information in automation API types
```
Apologies for the nitpick -- generally we try to write changelog entries in the present tense and imperative voice.